assisted-events-scrape/events/events_exporter.py

```diff
@@ -69,6 +69,16 @@ def _get_all_docs(self, stream: EventStream, offsets: DateOffset) -> Iterable[di
                                 query=query, request_timeout=DEFAULT_TIMEOUT)
             all_docs = chain(all_docs, docs)
 
+        # if it is unpartitioned data, but with offset
+        if offsets.size() > 0 and not stream.options.partition_key:
+            for partition, offset in offsets.getAll().items():
+                # partition should be None
+                query = self._get_query(stream, partition, offset)
+                log.debug(f"Not first time retrieving non-partitioned stream {stream.name} (query: {query})")
+                docs = helpers.scan(self._es_client, index=stream.name, size=self._config.chunk_size,
+                                    query=query, request_timeout=DEFAULT_TIMEOUT)
+            all_docs = chain(all_docs, docs)
```

Comment:
shouldn't it be in the for loop as `docs` is assigned in it?

Reply:
Oops, you're right. It will never be looping as it is unpartitioned, but you're right